Tasks/Vadeika/Task1.py

```diff
@@ -0,0 +1,12 @@
+"""Task1."""
+
+import decimal
+
+init_dep = decimal.Decimal(20000)
```

Comment:
just a small notice - it's better to create the decimal values from string literals instead of any other (int is fine, but float's not so much)